server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -22,6 +23,10 @@ class AdminPredicateConfiguration {
       return
     }
 
+    addEventListenerToElements('button[type=submit]', 'click', (event: Event) =>
```

Comment:
**optional:** might be better to use an ID to identify the button in case the UI changes in the future

Reply:
Good call - updated!